09. cli-and-http-request/clihelper/WriteUsingErrortLoggers.go

```diff
@@ -0,0 +1,8 @@
+package clihelper
+
+func WriteUsingErrortLoggers(c CliBindingProperties, d []byte) {
+	_, err := w.Write(d)
+	if err != nil {
```

Comment:
Here we have to write for all error loggers.

Let's rename this `c CliBindingProperties` to `cliBindingProperties`
Let's rename this `d []byte` to `line string`

```golang
for _, errorLogger = cliBindingProperties.WritersCollection.ErrorLoggers {
     errorLogger.Writer(cliBindingProperties.config, line)
}
```

Same goes for `WriteUsingLoggers`

Reply:
Done